config.md

```diff
@@ -671,6 +670,16 @@ Many Redis installations do not use this field.
 - Type: `string`
 - Environment variable: `REFINERY_REDIS_PASSWORD`
 
+### `AuthCode`
```

Comment:
Hey @kentquirk , I'm not sure if this is just some missing doc that didn't generate before?

Reply:
Yes; from a contributed PR -- I knew we were going to be regenerating docs several times before the release, so I didn't make them do it.